docs/how-to/change-log-level.rst

```diff
@@ -0,0 +1,34 @@
+===============================
+Change log level in MicroCeph
+===============================
+
+By default, the MicroCeph daemon runs with the log level set to DEBUG. While that is the desirable
+behaviour for a good amount of use cases, there are instances when this level is far too high -
+for example, embedded devices where storage is much more limited. For these reasons, the MicroCeph
+daemon exposes a way to both get and set the log level.
+
+Configuring the log level
+-------------------------
+
+MicroCeph includes the command `log`, with the sub commands `set-level` and `get-level`. When setting, we support both string and integer formats for the log level. For example:
```

Comment:
* double backticks for monospace (commands). single ones give italics in RST
* s/sub command/sub-command

(change everywhere in the document)

Reply:
done.